simulators/trin-bridge/Cargo.toml

```diff
@@ -5,9 +5,9 @@ authors = ["https://github.com/ethereum/portal-hive/graphs/contributors"]
 edition = "2021"
 
 [dependencies]
-ethportal-api = { git = "https://github.com/ethereum/trin" }
-portal-bridge = { git = "https://github.com/ethereum/trin" }
-trin-validation = { git = "https://github.com/ethereum/trin" }
+ethportal-api = { git = "https://github.com/ethereum/trin", rev = "b530dc3d40d51c21c800089eacb2852ebd8c4d45" }
+portal-bridge = { git = "https://github.com/ethereum/trin", rev = "b530dc3d40d51c21c800089eacb2852ebd8c4d45" }
```

Comment:
Does this mean we would need to update this `rev` every time we want to test the newer code of the bridge? I think the current (pre-PR) hive behavior is a good thing, that it always tests whatever trin & bridge is on master.

Reply:
The current code behavior wouldn't test that since we use Cargo.lock. If you want that behavior we would need to not specify rev and not use locks

Reply:
I think this test should be changed to use the portal-bridge as if it was a client like trin. Then it would be testing it how you think it should be done as well. So we would pull the docker image for the bridge.

So in short I think how the bridge test setup rn is not how you want it regardless. So I propose we

remove the portal bridge dependency in the Cargo.toml
Then use the bridge docker image as if it was a client. This will allow for better tests!

I can do that in the next PR and we can merge this as is.

Reply:
Yeah, I'm fine with that 👍🏻